docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,32 +1,99 @@
-# Forecast quarterly house sales using MindsDB
```

Comment:
Using Capitalized.

Reply:
@Ricram2 
I noticed that in the Home Rentals tutorial, the main header is used with capitalized first letters, so I adjusted the remaining ones (it was done in my yesterday's comitts). Please see the screenshots below.
![image](https://user-images.githubusercontent.com/109554435/181035205-e9174d55-2f9f-4c2a-9847-039842b8876d.png)
![image](https://user-images.githubusercontent.com/109554435/181035236-a3ba0082-05c9-4048-817a-4a720273f3c0.png)
![image](https://user-images.githubusercontent.com/109554435/181035254-9193a92a-8c32-4063-bdae-5864a36095f2.png)